### ISSUE TITLE
Troubleshoot CORS issue with assets cached in S3

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#56151]
# Description
Some of the content displayed in the preview environments is attempting to pull from a dev s3 bucket with our current configuration. This produces errors and incorrect display of the instance. How might we reconfigure the PE instances to pull assets without causing a CORS issue.

![image.png](https://images.zenhubusercontent.com/5d2633e0a6db635dbcbaf8d5/820cd216-b69d-46ae-816d-f82ef3922aa0)

# Acceptance Criteria
- [ ] All assets display correctly in the PE instance

**Comment by cyriltech** [comment]
I am implementing CORS policies https://github.com/department-of-veterans-affairs/devops/pull/12767. PR needs modification as terraform plan is not reflecting changes.

**Comment by pjhill** [comment]
This appears to be blocked by a Terraform issue. Attempting to apply the changes results in an aberrant behavior from Terraform.

We held several multi-hour huddles with DevOps engineers. The current thought is that we will need to update a minor version of terraform-provider-aws to [3.76.1](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v3.76.1).

This is currently blocking us from adding a CORS policy to the bucket that PEs are pointing to for certain assets.

**Comment by pjhill** [comment]
We tried working around this issue by adding a [new matrix entry to the content-build build](https://github.com/department-of-veterans-affairs/content-build/tree/ci_matrix_local_build). This didn't work because the build script fails when it attempts to create a symlink in the content-build build to vets-website resources. We tried logging out the content-build symlink step. It didn't work. I'm not sure why.

**Comment by cyriltech** [comment]
CORS policy succesfully applied for s3 bucket "vetsgov-website-builds-s3-upload![image.png](https://images.zenhubusercontent.com/6385292bddbc0cbb14b84a51/75f47266-5dc6-415e-acd8-99180fb58eb1)" However we are still getting errors fetching contents. Need a confirmation of the origin we are trying to fetch dependencies. An inspect is showing different origins.

**Comment by JoeTice** [comment]
Sprint 24 Status - In Progress, will be completed in Sprint 25 - We've been working with the team and some members of the DevOps COP to get this working, we believe we are nearly complete, but need to address the errors Cyril mentioned above.

**Comment by pjhill** [comment]
Summary of the current state --
* We originally had the wrong bucket. The correct bucket is: dev-va-gov-assets
* The correct bucket is not currently configured in Terraform.
* We reached out to the bucket owners to ask if we can configure it in Terraform. They said yes.
* Created a [new ticket](https://app.zenhub.com/workspaces/platform-tech-team-4-633b069d2920b776613c93d8/issues/gh/department-of-veterans-affairs/va.gov-team/57363) to configure this bucket in Terraform and add a CORS policy that resolves this issue.
